metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/PolarisConnectorTableService.java

```diff
@@ -149,7 +155,13 @@ public TableInfo get(final ConnectorRequestContext requestContext, final Qualifi
                 .getTable(name.getDatabaseName(), name.getTableName())
                 .orElseThrow(() -> new TableNotFoundException(name));
             final TableInfo info = polarisTableMapper.toInfo(polarisTableEntity);
+            final boolean isView = connectorContext.getConfig().isCommonViewEnabled()
```

Comment:
At this layer there shouldn't be logic to selectively enable views, it should always use view logic whenever the table is a view